runtime/src/registry.rs

```diff
@@ -415,6 +437,7 @@ decl_event!(
     pub enum Event {
         CheckpointCreated(CheckpointId),
         CheckpointSet(ProjectName, Id, CheckpointId),
+        MemberRegistered(Id, Id),
```

Comment:
Forgot this one! Since the types are ambiguous, I'd rather change this to:

```suggestion
        MemberRegistered { user_id: Id, org_id: Id },
```

Reply:
seems like the macro does not support variants with named fields :/